pkg/client/cli/cmd_list.go

```diff
@@ -163,7 +164,10 @@ func DescribeIntercept(ii *manager.InterceptInfo, volumeMountsPrevented error, d
 
 	fields = append(fields, kv{"Intercepting", func() string {
 		if ii.MechanismArgsDesc == "" {
-			return fmt.Sprintf("using mechanism=%q with args=%q", ii.Spec.Mechanism, ii.Spec.MechanismArgs)
+			if len(ii.Spec.MechanismArgs) > 0 {
+				return fmt.Sprintf("using mechanism=%q with args=%q", ii.Spec.Mechanism, ii.Spec.MechanismArgs)
```

Comment:
Should we also include the metadata that is added to the intercept in this output?